draft-dejong-remotestorage-06.txt

```diff
@@ -330,19 +331,19 @@ Internet-Draft              remoteStorage                  November 2015
        * 403 for all requests that have insufficient scope, e.g.
              accessing a <module> for which no scope was obtained, or
              accessing data outside the user's <storage_root>,
-       * 404 for all DELETE and GET requests to documents that do not
-             exist on the storage,
-       * 304 for a conditional GET request whose pre-condition
+       * 404 for all DELETE, GET and HEAD requests to documents that do 
+             not exist on the storage,
+       * 304 for a conditional GET request whose precondition
              fails (see "Versioning" below),
        * 409 for a PUT request where any folder name in the path
              clashes with an existing document's name at the same
              level, or where the document name coincides with an
              existing folder's name at the same level.
-       * 412 for a conditional PUT or DELETE request whose pre-condition
+       * 412 for a conditional PUT or DELETE request whose precondition
              fails (see "Versioning" below),
        * 507 in case the account is over its storage quota,
-       * 4xx for all malformed requests (e.g. foreign characters in the
-             path), as well as for all PUT and DELETE requests to
+       * 4xx for all malformed requests (e.g. unsupported characters in 
+             the path), as well as for all PUT and DELETE requests to
```

Comment:
Ah, we could also say 'reserved', and refer to http://tools.ietf.org/html/rfc3986#section-2.2 here.